src/views/SignUp.js

```diff
@@ -76,11 +76,29 @@ function SignUpBeforeRedirect({ updateUserAndRedirect }) {
   const { t } = useTranslation('signup');
   const [emailSignIn, setEmailSignIn] = useState(false);
   return (
-    <PageContentWrapper>
+    <PageContentWrapper style={{ maxWidth: '100%' }}>
       <Helmet>
         <title>{t('title')}</title>
       </Helmet>
-      <p>{t('content')} </p>
+      <p>{t('one')} </p>
+      <p>{t('two')} </p>
+      <p>
+        {' '}
+        <span style={{ fontWeight: '900' }}>{t('notice')}</span> {t(window.screen.width > 750 ? 'three' : 'three_mob')}
+      </p>
+      <p>
+        {t(window.screen.width > 750 ? 'four' : null)}{' '}
+        <a href="https://drive.google.com/file/d/1arCs67WnAwWU02KvSrw0Mo7EV_IO9ofz/view?usp=drivesdk">{t('link')}</a>{' '}
+      </p>
+      <p style={{ fontWeight: '900', width: '100vw' }}>{t('five')}</p>
+      <p style={{ fontWeight: '900', width: '100vw' }}>{t('six')} </p>
```

Comment:
I think it will be easier to use styled components in this page. You can take a look at `src/components/ProtestForm/ProtestForm.js` for example, it especially useful in reused styles